src/Chatwork.php

```diff
@@ -30,13 +30,15 @@ class Chatwork
     /**
      * Create an API client
      *
-     * @param string $apiToken
-     * @param array $options
+     * @param Auth $auth
+     * @param string|null $proxyUrl
      */
-    public function __construct(Auth $auth)
+    public function __construct(Auth $auth, string $proxyUrl = null)
     {
+        $baseUri = ($proxyUrl != null) ? $proxyUrl : static::API_URI.'/'.static::API_VERSION.'/';
+
```

Comment:
@TungPro anh ơi anh để params là `$options` với key `base_uri` đi
```suggestion
    public function __construct(Auth $auth, array $options = [])
    {
        $baseUri = $options['base_uri'] ?? static::API_URI.'/'.static::API_VERSION.'/';

```

Reply:
ok e. để a sửa theo hướng này